### ISSUE TITLE
Cascaded diagnostic when there is an error in a parameter default value

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#16355]
The following method declaration

```cs
    static void M(int x = NotFound) { }
```

produces the two errors

```none
error CS0103: The name 'NotFound' does not exist in the current context
error CS1750: A value of type '?' cannot be used as a default parameter because there are no standard conversions to type 'int'
```

The first error is correct. The second is a cascaded diagnostic that should be suppressed.

**Comment by jcouv** [comment]
I'm fixing this along with https://github.com/dotnet/roslyn/issues/19013

**Comment by jcouv** [comment]
Fixed by #20077
